test/extensions/quic_listeners/quiche/platform/quic_thread_impl.h

```diff
@@ -12,14 +12,19 @@
 
 #include "common/common/assert.h"
 
+#include "test/test_common/thread_factory_for_test.h"
+
 #include "absl/synchronization/notification.h"
 
 namespace quic {
 
 // A class representing a thread of execution in QUIC.
 class QuicThreadImpl {
 public:
-  QuicThreadImpl(const std::string& /*name*/) {}
+  QuicThreadImpl(const std::string& /*name*/) {
+    thread_factory_ = &Envoy::Thread::threadFactoryForTest();
```

Comment:
make thread_factory_ a reference and init in the initializer, per Envoy convention.

Reply:
done

Reply:
I refreshed a few times but this still doesn't look done to me; maybe this a github glitch?

Reply:
I made the change in the wrong branch. Now done in real.